README.md

```diff
@@ -8,9 +8,25 @@ Python software can use the provided library (`hwilib`). Software in other langu
 
 ## Prerequisites
 
-Python 3 is required. The libraries and udev rules for each device must also be installed.
+Python 3 is required. The libraries and udev rules for each device must also be installed. Some libraries will need to be installed
 
-Install all of the libraries using `pip` (in virtualenv or system):
+For Ubuntu/Debian:
+```
+sudo apt install libusb-1.0-0-dev libudev-dev
+```
+
+For macOS:
+```
+brew install libusb
+```
+
+This project uses the [Poetry](https://github.com/sdispater/poetry) dependency manager. HWI and its dependencies can be installed via poetry with:
+
+```
+poetry install
```

Comment:
`poetry install hwi` ?

Reply:
No, you run that from within the source tree itself. I've clarified this.